src/components/pages/LogoutPage.tsx

```diff
@@ -0,0 +1,70 @@
+import Link from '@mui/material/Link';
+import { NavLink } from 'react-router-dom'
+import { Container, Typography } from '@mui/material';
+import { useTranslation } from 'react-i18next';
+import viewerState from '../../state/ViewerState';
+import React, { useEffect, useState } from 'react'
+import { useNavigate } from 'react-router-dom';
+import axios from 'axios';
+
+interface LogoutPageProps {
+    isLoggedin: boolean
+}
+
+const LogoutPage: React.FC<LogoutPageProps> = ({ isLoggedin }) => {
+    const { t } = useTranslation();
+    const navigate = useNavigate();
+
+    const [errorMessage, setErrorMessage] = useState<string | null>('');
+
+    const handleLogout = async () => {
+        try {
+            if (isLoggedin) {
+                let response = await axios.post('http://localhost:8000/logout/', null, {
+                    headers: {
+                      Authorization: `Token ${localStorage.getItem('token')}`,
+                    },
+                });
+                if (response.status === 200) {
```

Comment:
if this is async, and we're awaiting should this block go into .then?

Reply:
You are right, I just moved it inside of .then. I checked in the other files and there was the same bug in the Register page.